scripts/NextJs/configs/sentry.client.config.js

```diff
@@ -8,6 +8,10 @@ const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
 
 Sentry.init({
   dsn: SENTRY_DSN || '___DSN___',
+  // We recommend adjusting this value in production, or using tracesSampler
+  // for finer control
```

Comment:
I think we could try to find a terser comment that delivers the same message.
There's already too much to read, and people tend to ignore walls of text.

```suggestion
    // Adjust this value in production, or use tracesSampler for greater control
```